packages/accessibility/src/behaviors/Button/buttonBehavior.ts

```diff
@@ -14,6 +14,8 @@ const buttonBehavior: Accessibility<ButtonBehaviorProps> = props => ({
     root: {
       role: props.as === 'button' ? undefined : 'button',
       tabIndex: props.as === 'button' ? undefined : 0,
+      disabled:
+        props.disabled || props.loading ? (props.as === 'button' ? true : undefined) : undefined,
```

Comment:
pls revert

Reply:
I've changed it differently. The button will have `disabled` if it's not a loading button. For the loading button the class is redefined as LoadingButton and `disabled` is not being set (and the validator validates that it's not set).